tests/unit/all.js

```diff
@@ -23,7 +23,9 @@ define([
 	"./ToasterMessage-insert-show-hide-remove",
 	"./Toggle",
 	"./Button",
+	"./Button-bidi",
```

Comment:
We could then have a unit/bidi/all.js that contains all of them to launch bidi test independently or exclude them all more easily (and tests/unit/all would reference tests/unit/bidi/all).